### PR TITLE
Add delayed actions

### PR DESCRIPTION
**Pull request by nhunzaker**
delay

### CODE REVIEW

File: shells/dev/target/boot.js [hunk 1]
```diff
@@ -25,6 +37,7 @@ class Repo extends Microcosm {
       register() {
         return {
           [ADD]: this.add,
+          [eventuallyAdd]: this.add,
```
Comment (benjtinsley) [inline shells/dev/target/boot.js]: why is this one camel case when the other keys are all upper?

**Review by benjtinsley — APPROVED** [review]

**Comment by nhunzaker** [comment]
Eh. Yes. These should be camel case. I have no excuse

**Comment by efatsi** [comment]
Looks awesome. Not sure if the gif is distorting the colors but yellow -> [red,green] seems like it would make more sense for open -> [reject,resolve] status representation

**Comment by nhunzaker** [comment]
@benjtinsley updated the action names to be consistent (camelCase). @efatsi updated the colors to be a stoplight. Not sure why I made `open` pink:

screen shot 2017-05-26 at 8 03 14 am

**Pull request merged**